src/pages/_layouts/DefaultLayout.vue

```diff
@@ -16,15 +16,19 @@ const { isDesktop } = useBreakpoints();
 /**
  * COMPUTED
  */
-const isHomePage = computed(() => route.path === '/');
+const isHomePage = computed(() => route.name === 'home');
+const isPortfolioPage = computed(() => route.name === 'portfolio');
 </script>
 
 <template>
   <div>
     <AppNav />
-    <template v-if="isHomePage">
+    <template v-if="isHomePage || isPortfolioPage">
       <StakingProvider>
-        <AppHero />
+        <AppHero
+          :showInvestments="isPortfolioPage"
+          :showLearnMoreBtn="isHomePage"
+        />
```

Comment:
AppHero was initially used on every page, hence why we named it AppHero. I think we now need to split it into two separate components and use them on each individual page. e.g. on the home page have a HomePageHero and on the portfolio page PortfolioPageHero, or something similar to this.

Two reasons for this:
1. Nesting inside StakingProvider above means all the staking data is loaded on the home page where it's not needed.
2. There are queries being run in AppHero that are not needed on the home page so again there is excessive fetching.

Reply:
Makes sense. I will split the Hero components.

Reply:
Alright. I've created PortfolioPageHero and HomePageHero components. AppHero is now composable component.